connectors/sqs/src/main/java/io/camunda/connector/inbound/model/message/SqsInboundMessage.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.inbound.model.message;
+
+import java.util.Map;
+import java.util.Objects;
+
+public class SqsInboundMessage {
+  private String messageId;
+  private String receiptHandle;
+  private String mD5OfBody;
```

Comment:
This looks weird. Consider the following?

```suggestion
  private String bodyMD5Checksum;
```